### DISCUSSION TITLE
How to display dates in a language specific date format?

### DISCUSSION THREAD

**Discussion by zynicum** [datenstrom/community#273] [Ask a question] [answered]
My website location/language is German. I know, when dates are displayed on a page they are converted to a language specific date format.
In the file system/snippets/content-blogpages.php I need the HTML-Tag `<time>.`
My idea:
`<time datetime="<?php echo $page->getDateHtml("published","Y-m-d") ?>"><?php echo $page->getDateHtml("published") ?></time>`
The generated output is:
`<time datetime="[2017-07-25]">25.07.2017</time>`
But I need (**without the brackets**):
`<time datetime="2017-07-25">25.07.2017</time>`
How can I do this?

**Comment by markseuffert** [comment, marked as answer]
Hi, it looks like the [API for developers](https://datenstrom.se/yellow/help/api-for-developers) made this hard... there is no point to deny this. 

I suggest to split date handling into two separate functions:

`<?php echo $page->getDateHtml("published") ?>` converted to a  language specific date

`<?php echo $page->getDateFormattedHtml("published", "Y-m-d") ?>` for a custom date, e.g. 2017-07-25
`<?php echo $page->getDateFormattedHtml("published", "d.m.Y") ?>` for a custom date, e.g. 25.07.2017
`<?php echo $page->getDateFormattedHtml("published", "F") ?>` for the month as text, e.g. January
`<?php echo $page->getDateFormattedHtml("published", "l") ?>` for the weekday as text, e.g. Monday

If this looks good to you, please try the latest core extension.

**Comment by zynicum** [comment]
It works perfectly, thank you
